ui/components/multichain/pages/send/components/quote-card/hooks/useEthFeeData.tsx

```diff
@@ -53,7 +53,10 @@ export default function useEthFeeData(gasLimit = 0) {
 
   return useMemo(() => {
     const ethGasFee = new Numeric(
-      networkAndAccountSupports1559 ? gasFee1559 : gasPriceNon1559,
+      // TODO does this need to be removed? Needs to be tested with non 1559 network
+      networkAndAccountSupports1559
+        ? gasFee1559
+        : `0x${String(gasPriceNon1559 ?? 0)}`,
```

Comment:
Can we use `addHexPrefix` here instead

Reply:
Side note but is this not a numeric value? We would need to change the base in the next parameter as well if it's a hex

Reply:
If you search for `GAS_PRICE_API_MOCK_RESULT` or `gasPricesCallbackMock` it looks like the prices are in decimals

The infura https://gas.api.infura.io/networks/1/gasPrices api might be deprecated tho bc I can't find docs on it